shared/coordinator/src/model.rs

```diff
@@ -33,11 +33,13 @@ unsafe impl ZeroableInOption for Model {}
     Deserialize,
     InitSpace,
     TS,
+    PartialEq,
 )]
 #[repr(C)]
 pub enum LLMArchitecture {
     HfLlama,
     HfDeepseek,
+    HfAuto,
```

Comment:
@crypto-vincent can you confirm that just adding an enum value here is not a breaking storage change? I would assume not but going from boolean-valued to more makes me slightly concerned

Reply:
Yep, adding a case to a field-less `enum` should not break the memory layout.

The only exception where would be if the `enum` had fields like so:

```rust
enum MyValue {
  CaseA(u8),
  CaseB(u64), // Adding this case would enlarge the enum's max size and this would break memory layout
```